sync/file-syncer-config.yaml

```diff
@@ -0,0 +1,16 @@
+group:
+  # Actions excluding releasability
+  - repos: |
+      kvmware/test-infra
+    files:
+      - source: sync/workflows/
+        dest: .github/workflows/
+        exclude: |
+          synced-releasability.yaml
```

Comment:
Some repos don't want actions
https://github.com/knative-sandbox/knobots/blob/main/actions-exclude.yaml

Some repos don't want certain actions
https://github.com/knative-sandbox/knobots/blob/main/actions-omitted.yaml

It would be good to see if this configuration makes it easy to support the above two uses cases (or if we're changing it to opt-in vs. opt-out)

I generally want to make the configuration obvious to new contributors who are creating new repos